e2e/etcd_spawn_cov.go

```diff
@@ -55,9 +46,48 @@ func spawnCmd(args []string) (*expect.ExpectProcess, error) {
 		ep.StopSignal = syscall.SIGTERM
 		return ep, nil
 	}
+
+	if args[0] == ctlBinPath {
+		covArgs, err := getCovArgs()
+		if err != nil {
+			return nil, err
+		}
+		apiVersion := os.Getenv("ETCDCTL_API")
+		// set ctl command args as a env var
```

Comment:
`// avoid test flag conflicts in coverage enabled etcdctl by putting flags in ETCDCTLCOV_ARGS `?